src/config/config.zig

```diff
@@ -2,31 +2,43 @@ const std = @import("std");
 
 const dns_seed = [:0]const u8;
 
-const DNS_SEEDS = [3]dns_seed{
-    "seed.bitcoin.sipa.be",
-    "seed.bitcoin.sprovoost.nl",
-    "seed.btc.petertodd.net",
-};
-
 /// Global configuration for the node
 ///
 /// This is loaded from the `bitcoin.conf` file
 /// Must be loaded before any other modules are used.
 /// Must be compatible with Bitcoin Core's `bitcoin.conf` format.
 pub const Config = struct {
-    allocator: std.mem.Allocator,
+    const Self = @This();
+    /// Protocol version
+    pub const PROTOCOL_VERSION: i32 = 70015;
 
+    /// Known network ids
+    pub const BitcoinNetworkId = struct {
+        pub const MAINNET: [4]u8 = .{ 0xf9, 0xbe, 0xb4, 0xd9 };
+        pub const REGTEST: [4]u8 = .{ 0xfa, 0xbf, 0xd5, 0xda };
+        pub const TESTNET3: [4]u8 = .{ 0x0b, 0x11, 0x09, 0x07 };
+        pub const SIGNET: [4]u8 = .{ 0x0a, 0x03, 0xcf, 0x40 };
```

Comment:
```suggestion
        pub const MAINNET= [_]u8{ 0xf9, 0xbe, 0xb4, 0xd9 };
        pub const REGTEST= [_]u8{ 0xfa, 0xbf, 0xd5, 0xda };
        pub const TESTNET3= [_]u8{ 0x0b, 0x11, 0x09, 0x07 };
        pub const SIGNET= [_]u8{ 0x0a, 0x03, 0xcf, 0x40 };
```

Reply:
I prefer to have it expressively typed. If someone inadvertently adds one byte to one of those, it will fail due to the specified array len.